src/components/SatelliteManifestPanel/SatelliteManifestPanel.tsx

```diff
@@ -408,7 +409,7 @@ const SatelliteManifestPanel: FunctionComponent<SatelliteManifestPanelProps> = (
                 {countManifests(data, searchValue) === 0 && data.length > 0 && (
                   <NoSearchResults clearFilters={clearSearch} />
                 )}
-                {!isFetching && data.length === 0 && <NoManifestsFound />}
+                {data?.length === 0 && user && <CreateManifestPanel user={user} />}
```

Comment:
Does this need to be an optional? Also, not too familiar with this code, but why is the check to make sure data isn't being fetched removed? Does that have to do with the optional?